docs/src/validator/validator_length_rule.md

```diff
@@ -0,0 +1,20 @@
+## Length Rule
+The lengths of properties in string format provided by `cobra.Command` can be controlled using the Length Rule.
+
+## How to use
+1. Create a length rule by providing the limits for each attribute you want to control
+```go
+myLengthRule := &rules.Length{
+    Limits: map[string]rules.Limit{
+        "Use":   {Min: 1, Max: 5},
+        "Short": {Min: 4, Max: 10},
+        "Long":  {Min: 5, Max: 20},
+        "Not":   {Min: 2, Max: 22},
+    },
+}
+```
+2. Use ValidateLength function to validate your cobra command with the rule created above. It takes a verbose bool for enabling/disabling debug StatusLogs
```

Comment:
Each rule needs to be called explicitly?

My take is that end user api should finish in 2 lines no matter how many rules we have
```
validator := Validator{DefaultRuleConfig}
validator.vakidate(root)
```

Alternatively users should be able to override rule comfig.

Reply:
#### Now
```go
var lenValidator rules.Length // user can give their config here or use default like this
lenValidator.ValidateLength(cmd, true)
```
Is the above fine?
> Each rule needs to be called explicitly?

As rule is a separate package now, we might be able to figure out something when we do ruleExecutor